### PR TITLE
CAL-118 Fix ipv6 videographer validation

### PR DESCRIPTION
**Pull request by brendan-hofmann**
#### What does this PR do?
Fix scoped ipv6 addresses in videographer tokens.
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@clockard 
@glenhein 
@harrison-tarr 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@pklinef
@stustison
#### Any background context you want to provide?
This is a port of DDF-2416 
#### What are the relevant tickets?
CAL-118
#### Checklist:
- [ ] Documentation Updated
- [x] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: catalog/video/video-security/src/test/java/org/codice/alliance/video/security/validator/videographer/TestVideographerValidator.java [hunk 1]
```diff
@@ -194,4 +214,15 @@ public void testCanValidateBadToken() {
...
                 .getState(), is(ReceivedToken.STATE.INVALID));
     }
+
+    @Test
+    public void testCanValidateIpv6ReachabilityToken() {
+        TokenValidatorParameters params = new TokenValidatorParameters();
+        params.setToken(receivedTokenIpv6Reachability);
+        TokenValidatorResponse response = validator.validateToken(params);
+
+        assertEquals(ReceivedToken.STATE.VALID,
```
Comment (bdeining) [inline catalog/video/video-security/src/test/java/org/codice/alliance/video/security/validator/videographer/TestVideographerValidator.java]: Should use `assertThat` here

**Comment by pklinef** [comment]
:+1:

**Comment by peterhuffer** [comment]
LGTM

**Comment by glenhein** [comment]
👍

**Comment by brendan-hofmann** [comment]
Passed hero build.

**Comment by clockard** [comment]
👍

**Pull request merged**